sympy/solvers/inequalities.py

```diff
@@ -204,7 +204,7 @@ def reduce_rational_inequalities(exprs, gen, relational=True):
     >>> from sympy import Poly, Symbol
     >>> from sympy.solvers.inequalities import reduce_rational_inequalities
 
-    >>> x = Symbol('x', real=True)
+    >>> x = Symbol('x', extended_real=True)
```

Comment:
On master this doctest gives:
```julia
In [1]: from sympy.solvers.inequalities import reduce_rational_inequalities                                                       

In [2]: x = Symbol('x', real=True)                                                                                                

In [3]: reduce_rational_inequalities([[x + 2 > 0]], x)                                                                            
Out[3]: -2 < x ∧ x < ∞
```
However this PR gives:
```
In [3]: reduce_rational_inequalities([[x + 2 > 0]], x)                                                                            
Out[3]: -2 < x
```
This is because of x is real then x>oo auto-evaluates to true. I've changed it to extended_real=True so that the test output stays the same.

Would it be better to change the output for these tests (there are also examples)?

Reply:
or just include both tests so it is clear that we know they are different and are ok with that

Reply:
We can but I don't really know how to explain this to a user: why when x is declared as extended_real (and therefore possibly infinite) does the solution come back explicitly excluding oo?

Reply:
I've changed it to real=True and included an example with extended_real=True.